trestle/common/list_utils.py

```diff
@@ -79,7 +79,33 @@ def delete_item_from_list(item_list: List[Any], value: Any, key: Callable[[Any],
     return item_list
 
 
+def get_item_from_list(
+    item_list: Optional[List[Any]], value: Any, key: Callable[[Any], Any], remove: bool = False
+) -> Any:
+    """Get item from list if present."""
+    if not item_list:
+        return None
+    keys = [key(item) for item in item_list]
+    item = None
+    if value in keys:
+        index = keys.index(value)
+        item = item_list[index]
+        if remove:
+            del item_list[index]
+    return item
+
+
+def pop_item_from_list(item_list: Optional[List[Any]], value: Any, key: Callable[[Any], Any]) -> Any:
+    """Pop an item from a list if it is present based on the key."""
+    return get_item_from_list(item_list, value, key, True)
+
+
 def delete_list_from_list(item_list: List[Any], indices: List[int]) -> None:
     """Delete a list of items from a list based on indices."""
     for index in sorted(indices, reverse=True):
         del item_list[index]
+
+
+def merge_dicts(dest: Optional[Dict[str, str]], src: Optional[Dict[str, str]]) -> Dict[str, str]:
```

Comment:
Should we follow a convention of bash commands and do src, dest instead of dest, src?

Reply:
For me this is most natural and it's what I encounter more.  Either way can be confusing - but it is documented.